{"name": "test2_login_with_error_password", "status": "passed", "attachments": [{"name": "stdout", "source": "9781482b-f671-4beb-996c-6f4892360f8a-attachment.txt", "type": "text/plain"}], "start": 1689849669298, "stop": 1689849686141, "uuid": "c64a38db-3814-4ebf-a4b5-df69e27dabc3", "historyId": "b63947ce1f51a1679554edf642e2875b", "testCaseId": "b63947ce1f51a1679554edf642e2875b", "fullName": "testcase.login_logout.test_login_with_wrong_info.TestErrorLogin#test2_login_with_error_password", "labels": [{"name": "severity", "value": "minor"}, {"name": "feature", "value": "Login error"}, {"name": "story", "value": "login with error password"}, {"name": "parentSuite", "value": "testcase.login_logout"}, {"name": "suite", "value": "test_login_with_wrong_info"}, {"name": "subSuite", "value": "TestErrorLogin"}, {"name": "host", "value": "Joys-MacBook-Pro.local"}, {"name": "thread", "value": "51806-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.login_logout.test_login_with_wrong_info"}]}